rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -207,6 +207,7 @@ def constant_value_to_py(type_, value):
 
 def escape_string(s):
     s = s.replace('\\', '\\\\')
+    s = s.replace('"', '\\"')
```

Comment:
fair enough, fixed in https://github.com/ros2/rosidl/pull/204/commits/1e7941a2808378e5f277f54838320db23cececde